Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -994,6 +996,7 @@ This allows the WebView to interpret JavaScript. It should be enabled only if ne
 
 - the communication to the endpoints consistently relies on HTTPS (or other protocols that allow encryption) to protect HTML and JavaScript from tampering during transmission
 - JavaScript and HTML are loaded locally, from within the app data directory or from trusted web servers only.
+- The user cannot define which sources to load by means of loading different resources based on a user provided input.
```

Comment:
```suggestion
- It is not possible to navigate the WebView to a specified URL through app-to-app communication without properly validating the URL.
- It is not possible to navigate to untrusted sources by means of interacting with the WebView.
```
Modified it somewhat. Allowing a user to choose which page to load seems ok at first glance.
Also added one other scenario. This one actually has a preventive measure, so maybe that's already tackled somewhere else?

Reply:
This does not make sense: there are many type of ways how a user can load that resource: IPC, wrong input within the app itself, so closing this one :)